pom.xml

```diff
@@ -60,14 +54,32 @@
 			<artifactId>vertx-web-templ-handlebars</artifactId>
 			<version>3.3.3</version>
 		</dependency>
-        <dependency>
-            <groupId>org.mockito</groupId>
-            <artifactId>mockito-core</artifactId>
-            <version>2.9.0</version>
-            <scope>test</scope>
-        </dependency>
-
+		
+		<!-- Dependencies only needed for testing: -->
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
```

Comment:
Is there a reason why have choosen these old versions? Current is:

* JUnit: 5.2.0 (but with junit-vintage-engine or, if we want, junit-jupiter-engine AKA JUnit 5)
* Mockito: 2.19.0
* Selenium: 3.13.0
* Guava: 25.1-jre